src/DataViews/FunctionsDataView.cpp

```diff
@@ -76,8 +79,8 @@ bool FunctionsDataView::ShouldShowFrameTrackIcon(AppInterface* app, const Functi
          app->HasFrameTrackInCaptureData(instrumented_function_id.value());
 }
 
-std::string FunctionsDataView::BuildSelectedColumnsString(AppInterface* app,
-                                                          const FunctionInfo& function) {
+std::string FunctionsDataView::BuildSelectedAndFrametrackIconsString(AppInterface* app,
```

Comment:
I'm not sure I would refer to these as "icons" since they are characters. You could also just drop the "Icons" part.